.ci/test.sh

```diff
@@ -104,7 +104,14 @@ if [[ $TASK == "swig" ]]; then
     exit 0
 fi
 
-conda install -q -y -n $CONDA_ENV cloudpickle dask distributed joblib matplotlib numpy pandas psutil pytest scikit-learn scipy
+# temporary fix for https://github.com/microsoft/LightGBM/issues/4285
+if [[ $PYTHON_VERSION == "3.6" ]]; then
+    DASK_DEPENDENCIES="dask distributed"
+else
+    DASK_VERSION="dask=2021.4.0 distributed=2021.4.0"
```

Comment:
CI tests passed because they are skipped in case Dask is not installed.
```suggestion
    DASK_DEPENDENCIES="dask=2021.4.0 distributed=2021.4.0"
```

Reply:
ah!!! Sorry, thank you very much for catching that.